provider/cmd/pulumi-resource-aws/main.go

```diff
@@ -22,15 +22,19 @@ import (
 
 	aws "github.com/pulumi/pulumi-aws/provider/v5"
 	pf "github.com/pulumi/pulumi-terraform-bridge/pf/tfbridge"
-	"github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfbridge"
+	//"github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfbridge"
 )
 
 //go:embed schema-embed.json
 var pulumiSchema []byte
 
+//go:embed bridge-metadata.json
+var bridgeMetadata []byte
```

Comment:
It think this bit was really needed to get past an error, since the PF provider needs to read renames data generated by tfgen. Error msgs need work unfortunately.